### PR TITLE
feat: add neo-tree support

### PR DESCRIPTION
**Pull request by cseickel**
This update will enable integration with neo-tree if it is installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,16 +20,32 @@ https://user-images.githubusercontent.com/14187674/211327507-39f21a74-0a43-43f0-
...
 
+### For Nvim-tree Users
 ```lua
 use {
   'antosha417/nvim-lsp-file-operations',
   requires = {
-    { "nvim-lua/plenary.nvim" },
-    { "kyazdani42/nvim-tree.lua" },
+    "nvim-lua/plenary.nvim",
+    "nvim-tree/nvim-tree.lua",
```
Comment (gegoune) [inline README.md:29]: Thanks buddy! :)

Comment (cseickel) [reply README.md:29]: No problem!

**Comment by dmmulroy** [comment]
Would love to see this addition - thanks @cseickel!

**Comment by gungun974** [comment]
Personally before discovering the `typescript-tools.nvim` project, I didn't know this project at all but I really like it.

I hope this PR will quickly merge to be able to take full advantage of the automatic renaming of imports ^^

Thanks @cseickel for the support ! :purple_heart:

**Comment by antosha417** [comment]
@cseickel thank you a lot for your contribution 🔥

**Comment by antosha417** [comment]
@cseickel might be good to add a link to this plugin to neo-tree readme or wiki. So there will be a way for people to learn about your awesome contribution

**Pull request merged**